x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/services/elastic/ElasticInferenceService.java

```diff
@@ -133,6 +137,19 @@ private static Map<String, DefaultModelConfig> initDefaultEndpoints(
                     elasticInferenceServiceComponents
                 ),
                 MinimalServiceSettings.chatCompletion()
+            ),
+            DEFAULT_ELSER_2_MODEL_ID,
```

Comment:
How come this was missing - does this mean EIS-based ELSER was never available in 9.0?